character-controller.js

```diff
@@ -80,20 +82,20 @@ function makeCancelFn() {
 const heightFactor = 1.6;
 const baseRadius = 0.3;
 function loadPhysxCharacterController() {
-  const avatarHeight = this.avatar.height;
-  const radius = baseRadius/heightFactor * avatarHeight;
-  const height = avatarHeight - radius*2;
+  const avatarHeight = this.avatar?.height || 1;

```

Comment:
If there is no avatar then this will create an incorrect collider since you don't know the size.

I can see why players would be initialized without avatars, but that means when the avatar pops in we need to set the capsule size correctly (we may need to remake it).

Reply:
If this is handled, please link the place where it's handled.

If not it needs to be handled.

Reply:
This ordering problem probably related to why character starting below floor when landing in world. Will look into it.